astropy/io/fits/convenience.py

```diff
@@ -1080,7 +1080,7 @@ def _stat_filename_or_fileobj(filename):
     noexist_or_empty = ((name and
                          (not os.path.exists(name) or
                           (os.path.getsize(name) == 0)))
-                         or (not name and loc == 0))
+                        or (not name and loc == 0))
```

Comment:
make it consistent, binary operators should be on the same side for all, shouldn't they? also, extra indent after the linebreak, before the operator is  nice (as it's actually done below in the next file)

Reply:
Probably, but that's a work for a dedicated tool, there are too many of those and it is impossible to be consistent where there is more than one developer.